io.c

```diff
@@ -9446,8 +9452,10 @@ rb_io_set_encoding_by_bom(VALUE io)
  *
  *  Examples:
  *
- *    File.new('t.tmp', autoclose: true)
- *    File.new('t.tmp', internal_encoding: nil)
+ *    f = File.new('t.tmp', autoclose: true)
+ *    f.close
+ *    f = File.new('t.tmp', internal_encoding: nil)
+ *    f.close
```

Comment:
I'm not sure if there's a lot of value for these examples since they just show how to use keyword arguments (and doesn't show what the significance is).

Reply:
Omitted.